manager/integration/tests/test_basic.py

```diff
@@ -280,6 +280,12 @@ def backup_test(client, host_id, volname):
     setting = client.update(setting, value=common.get_backupstore_url())
     assert setting["value"] == common.get_backupstore_url()
 
+    if if_s3_backup(setting["value"]):
```

Comment:
`is_backupTarget_s3` sounds better.
1. Don't use `if` as the prefix of a function. Use `is` or `not` instead. Because `if if` stutters.
2. We may have e.g. `is_backupTarget_rancher` later.